go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -485,3 +512,14 @@ func newRatelimiter(qps int) workqueue.RateLimiter {
 		&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(qps), qps*5)},
 	)
 }
+
+func (c *Controller) reconcileOVNLBs(stopCh <-chan struct{}, lbCache *ovnlb.LBCache) {
+	for {
+		select {
+		case lb := <-c.ovsdbQueue:
+			lbCache.AddOrUpdateEntry(lb.Name, lb.UUID)
```

Comment:
I guess that this going to be called for all the LBs on each reconnect? Just a consideration, not actually worried about it.

If we ever suspect that there is a reason for the dup LBs to be different in more than just the UUID, we might have to trigger a service sync. 

There is code that shortcuts the service sync if there is not actual difference on the LBs to be applied:

https://github.com/ovn-org/ovn-kubernetes/blob/2be204b206363ad1204919670354ad7b78d48c4c/go-controller/pkg/ovn/controller/services/services_controller.go#L318-L320

Just be aware the we will be a bit lazy on how we resolve the duplicates but that should not be a problem if they are actual duplicates except for the UUID, right?

Reply:
Yeah all lbs will be added on reconnect.

yeah, right now it takes a change to the service to trigger duplicate detection. I'm going to add some more code to be more proactive here and attempt to add the key to the workqueue so it will be soon processed by the controller.